config/finders/all_content.yml

```diff
@@ -6,16 +6,12 @@ locale: en
 publishing_app: rummager
 rendering_app: finder-frontend
 schema_name: finder
-title: All content
```

Comment:
Does it still work without a title? You might need to update the content schemas (either making title optional or adding a new field to specify whether the title should be displayed).

https://github.com/alphagov/govuk-content-schemas/blob/1c942c7e6f5101e69a563024630d2d27ce3de092/formats/finder.jsonnet

https://github.com/alphagov/govuk-content-schemas/blob/1c942c7e6f5101e69a563024630d2d27ce3de092/formats/shared/default_format.jsonnet#L7

Reply:
ah I did not know that. I'll add an empty field instead

Reply:
That does not work either. I put the title back for now. Removing the title is now a new card